### ISSUE TITLE
Genius to Singular is not giving appropriate result

### ISSUE THREAD

**Issue by pujansrt** [spencermountain/compromise#454]
genius to singular is providing wrong output "geniu"

```
const r = nlp('geniuses').nouns().toSingular().out('text');
console.log(r); // geniuse
```
Expected Result: genius

```
const r = nlp('genius').nouns().toSingular().out('text');
console.log(r); // geniu
```
Expected Result: genius

**Comment by spencermountain** [comment]
thanks Pujan, will take a look

**Comment by spencermountain** [comment]
hey, this is fixed in `11.5.0`
thanks!
